### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by risqi2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,5 @@ commit a
 commit b
 commit c
```
Comment (risqiikhsani) [inline README.md:24]: add something

File: README.md [hunk 2]
```diff
@@ -23,3 +23,5 @@ commit a
...
 
+changed
```
Comment (risqiikhsani) [inline README.md:26]: add something

**Review by risqiikhsani — CHANGES_REQUESTED** [review]
update it

File: README.md (hunk 1 above)
Comment (risqi2) [reply README.md:24]: done

File: README.md (hunk 2 above)
Comment (risqi2) [reply README.md:26]: done

**Review by risqiikhsani — APPROVED** [review]

**Pull request merged**